bundle.yaml

```diff
@@ -1,25 +1,26 @@
 series: trusty
 services:
   filebeat:
-    charm: "cs:~containers/trusty/filebeat-5"
+    charm: "cs:trusty/filebeat-1"
     annotations:
       "gui-x": "808.188"
       "gui-y": "426.512"
   topbeat:
-    charm: "cs:~containers/trusty/topbeat-8"
+    charm: "cs:trusty/topbeat-1"
     annotations:
       "gui-x": "819.045"
       "gui-y": "585.538"
   kibana:
-    charm: "cs:~containers/trusty/kibana-1"
+    charm: "cs:trusty/kibana-12"
     num_units: 1
+    expose: true
```

Comment:
Looks like i missed a push. I have removed the artifact and added that option. 1 sec and i'll get this addressed